components/admin_console/admin_definition.jsx

```diff
@@ -2389,6 +2389,21 @@ const AdminDefinition = {
                             it.stateIsFalse('LdapSettings.EnableSync'),
                         ),
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_TEXT,
+                        key: 'LdapSettings.GuestFilter',
+                        label: t('admin.ldap.guestFilterTitle'),
+                        label_default: 'Guest Filter:',
+                        help_text: t('admin.ldap.guestFilterFilterDesc'),
+                        help_text_default: '(Optional) Enter an AD/LDAP filter to use when searching for guest objects. Only the users selected by the query will be able to access Mattermost as Guests. Guests are prevented from accessing teams or channels upon logging in until they are assigned a team and at least one channel.',
+                        placeholder: t('admin.ldap.guestFilterEx'),
+                        placeholder_default: 'E.g.: "(objectClass=guests)"',
+                        isDisabled: it.both(
+                            it.stateIsFalse('LdapSettings.Enable'),
+                            it.stateIsFalse('LdapSettings.EnableSync'),
```

Comment:
You don't need to have the sync enable to allow guests to log in as guests, I think is not needed. see the other comment.

Reply:
@jespino These are the same setting as the `UserFilter`.  It is only disabled if BOTH Enable AND EnableSync are false.  So, no, you don't need to have sync enabled to allow guest login.